### DISCUSSION TITLE
en/blog/introducing-gironajs-the-javascript-community-of-girona

### DISCUSSION THREAD

**Discussion by giscus** [gironajs/gironajs.com#25] [Announcements]
# en/blog/introducing-gironajs-the-javascript-community-of-girona

GironaJS is a Girona-based community of JavaScripters. At GironaJS, we do blogging in Catalan, monthly talks, workshops, and many other things related to JavaScript.

https://gironajs-com-git-add-blog-comments-gironajs.vercel.app/en/blog/introducing-gironajs-the-javascript-community-of-girona

**Comment by llorenspujol** [comment]
very nice blog! 👏
